RecoPixelVertexing/PixelTrackFitting/src/PixelTrackReconstruction.cc

```diff
@@ -77,7 +77,11 @@ void PixelTrackReconstruction::run(TracksWithTTRHs& tracks, edm::Event& ev, cons
           continue;
         }
       }
-
+      // all legacy tracks are "highPurity"
+      // setting all others bits as well (as in ckf)
+      track->setQuality(reco::TrackBase::loose);
```

Comment:
I can roll back these changes to see if the regression in HLT-tau disappear.
Even if it does something strange is going on in there.